src/views/product-downloads-view/index.tsx

```diff
@@ -83,25 +84,23 @@ const ProductDownloadsView = ({
           </Text>
         </div>
       </div>
-      {
-        <>
-          <label style={{ display: 'block' }}>Version (temp switcher)</label>
-          <select
-            onChange={(e) => setSelectedVersion(e.target.value)}
-            value={selectedVersion}
-          >
-            {versionSwitcherOptions.map((option) => (
-              <option key={option.value} value={option.value}>
-                {option.label}
-              </option>
-            ))}
-          </select>
-        </>
-      }
-      <p>
-        This content should only show when the{' '}
-        <code>enable_new_downloads_view</code> flag is on
-      </p>
+      <div style={{ marginBottom: 48 }}>
+        <label style={{ display: 'block' }}>Version (temp switcher)</label>
+        <select
+          onChange={(e) => setSelectedVersion(e.target.value)}
+          value={selectedVersion}
+        >
+          {versionSwitcherOptions.map((option) => (
+            <option key={option.value} value={option.value}>
+              {option.label}
+            </option>
+          ))}
+        </select>
+      </div>
+      <DownloadsSection
+        packageManagers={pageContent.packageManagers}
+        selectedRelease={releases.versions[selectedVersion]}
+      />
```

Comment:
This is the main change in this file other than adding a `<div>` around the temp switcher for styling purposes.